k8s/apis/solution/v1/solution_webhook.go

```diff
@@ -76,6 +79,9 @@ func (r *Solution) Default() {
 				r.Labels = make(map[string]string)
 			}
 			r.Labels["rootResource"] = r.Spec.RootResource
+			if projectConfig.UniqueDisplayNameForSolution {
```

Comment:
We need to do the same check at API layer for the case where REST API is invoked directly without K8s.

Reply:
This PR is to remove the unique name check by config. But I didn't find any unique name checks in the API code. So there are no changes in API side.